src/modules/launchUI.js

```diff
@@ -0,0 +1,81 @@
+import InvolvementUI from './InvolvementUI.js';
+import LaunchService from './LaunchService.js';
+
+const launchCardTemplate = document.querySelector('.launch-card.template');
+const launchCount = document.querySelector('.launch-count');
+const launchList = document.querySelector('.launch-list');
+const launchModal = document.getElementById('launch-modal');
+const img = launchModal.querySelector('.launch-img');
+const h1 = launchModal.querySelector('h1');
+const lsp = launchModal.querySelector('.lsp');
+const mission = launchModal.querySelector('.mission');
+const pad = launchModal.querySelector('.pad');
+const location = launchModal.querySelector('.location');
+const btnClose = document.querySelector('.close');
+btnClose.addEventListener('click', () => LaunchUI.closeModal());
+
+export default class LaunchUI {
+  static renderLaunches = () => {
+    LaunchService.getLaunches().then((launches) => {
+      launchCount.innerText = LaunchService.getLauchCount(launches);
```

Comment:
- [ ] I think it's better to create a specific function to calculate the launches counter. A counter function should look for specific DOM elements and make the counting based on what is actually displayed on the page. Don't forget these count functions need to be covered with unit tests using Jest. Don't forget the case when the count is zero.